Resources/Prototypes/Loadouts/role_loadouts.yml

```diff
@@ -553,6 +553,7 @@
   - GroupTankHarness
   - PsychologistJumpsuit
   - MedicalBackpack
+  - PsychologistBelt
```

Comment:
move it to where the others are below and change the comments to a single begin end pair like in CONTRIBUTING.md